safe/common/parameters/resource_parameter.py

```diff
@@ -10,8 +10,8 @@
 if PARAMETERS_DIR not in sys.path:
     sys.path.append(PARAMETERS_DIR)
 
-from safe_extras.parameters.float_parameter import FloatParameter
-from safe_extras.parameters.unit import Unit
+from parameters.float_parameter import FloatParameter
+from parameters.unit import Unit
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->